lms/djangoapps/courseware/courses.py

```diff
@@ -135,8 +136,12 @@ def course_image_url(course):
         else:
             url += '/images/course_image.jpg'
     else:
-        loc = StaticContent.compute_location(course.id, course.course_image)
-        url = StaticContent.serialize_asset_key_with_slash(loc)
+        try:
+            loc = StaticContent.compute_location(course.location.course_key, course.course_image)
+            url = StaticContent.serialize_asset_key_with_slash(loc)
+        except InvalidKeyError:
+            # handler for split courses, if course_image is blank
```

Comment:
Should we just check if `course.course_image` is blank? Are there other `course_image` values that could cause an InvalidKeyError? If so, we should add tests for them. If not, we probably can add another condition just to check if the `course.course_image` is blank.